models/image_segmentation_efficientsam/README.md

```diff
@@ -0,0 +1,37 @@
+# image_segmentation_efficientsam
+
+EfficientSAM: Leveraged Masked Image Pretraining for Efficient Segment Anything
+
+Notes:
+- 
```

Comment:
The original repo offers EfficientSAM-S and -Ti. Which one is used here? Add a note to describe including version (shasum, md5sum, etc.). Also describe how you convert the model to ONNX (a script would be nice).

Also need to describe how many clicks are required.